guides/nodejs.md

```diff
@@ -0,0 +1,308 @@
+# Node.js Developer's Guide
+
+When creating a Node.js function using the `func` binary, the project directory
+looks like a typical Node.js project. Both HTTP and Event functions have the same
+template structure.
```

Comment:
I'd add the specific command for function creation somewhere in this section.

Eg. to create Node.js function you can run this command:
```
❯ func create
```